torch/csrc/api/include/torch/data/init.h

```diff
@@ -0,0 +1,171 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <torch/data/datasets/chunk.h>
+namespace py = pybind11;
+
+namespace torch {
+namespace data {
+
+void init_dataset_bindings(PyObject* module);
+void init_dataset_bindings_impl(PyObject* module);
+void init_dataset_bindings_test(PyObject* module);
+
+namespace samplers {
+class SamplerWrapper : public Sampler<> {
+ public:
+  explicit SamplerWrapper(
+      std::shared_ptr<Sampler<>> sampler,
+      size_t total_stride = 0,
+      size_t current_stride = 0)
+      : sampler_(std::move(sampler)),
+        current_stride_(current_stride),
```

Comment:
Maybe we can consider adding some comment explaining what current_stride_ and total_stride_ do? It might be confusing for people first look at it.

Reply:
Great idea. Adding documentation to SamplerWrapper class, which includes strides